python/tvm/relay/frontend/tensorflow.py

```diff
@@ -676,6 +676,17 @@ def _impl(inputs, attr, params):
                                  'Taxis', '_class'])(new_input, attr)
     return _impl
 
+def _gather_v1():
```

Comment:
Both versions share almost same logic across.
Advice to differ on axis based on inputs count or an argument.